test/model/calculations.test.ts

```diff
@@ -10,56 +10,78 @@ describe(Calculations.OutstandingStockSharesCalculator, () => {
 
   test("issuances", () => {
     const subject = new Calculations.OutstandingStockSharesCalculator();
-    subject.apply(fakeStockTxn("ISSUANCE", { quantity: "10" }));
-    subject.apply(fakeStockTxn("ISSUANCE", { quantity: "1" }));
+    subject.apply(fakeStockTxn("STOCK_ISSUANCE", { quantity: "10" }));
```

Comment:
**Non-blocking:** If you're going to make this change, shouldn't you also rename the function to `fakeTxn`?